client/src/components/parent-signup-form/parent-signup.js

```diff
@@ -14,11 +15,19 @@ export class ParentSignup extends Component {
 
   handleSubmit = event => {
     event.preventDefault();
-    const data = JSON.stringify(this.state);
-    console.log("(Fake) submitting data: ", data);
+
+    const addParent = async () => {
+      try {
+        return await
+        axios.post('http://localhost:8080/parent/create', this.state)
```

Comment:
Give it more generic URL

Reply:
I believe I have fixed it now.  I have re-submitted this pull request.  Thank you!